### DISCUSSION TITLE
About use http2 to request  League of Legends Client Api get some question.

### DISCUSSION THREAD

**Discussion by xiaowuyaya** [encode/httpx#2098] [Q&A] [answered]
the code like this:
``` python 
import httpx

client = httpx.Client(http2=True, verify=False)
response = client.get('https://riot:ITdf5fK6X4aiMoXO2iXHCA@127.0.0.1:9558/lol-match-history/v3/matchlist/account/4124190404?begIndex=0&endIndex=20')
print(response.http_version)
print(response.json())
```

and i get error h2.exceptions.ProtocolError: cannot receive data before headers.
what it mean

**Comment by lovelydinosaur** [comment, marked as answer]
(What does any of it mean really??)

You'd see that exception if the server had a broken HTTP/2 implementation, and sent the response body without first sending the response status code and headers.

I'd suggest switching to plain ol' HTTP/1.1. Our reliable and trusty friend.

**Reply by xiaowuyaya** [reply]
I agree with you, it's normal if I request other h2 services.The problem is most likely in the lcu api, but I have to base it.
Fortunately, now I use the got package on nodejs to achieve my needs. Thank you for your reply and wish you a happy life ^o^

**Comment by Dobedoooo** [comment]
I use got in node too, and i cant access the same endpoint ```/lol-match-history/v3/matchlist/account/``` since yesterday.
did you meet the same problem?
